code/game/gamemodes/antag_hud.dm

```diff
@@ -10,8 +10,6 @@
 		CRASH("join_hud(): [M] ([M.type]) is not a mob!")
 	if(M.mind.antag_hud && !slave) //note: please let this runtime if a mob has no mind, as mindless mobs shouldn't be getting antagged
 		M.mind.antag_hud.leave_hud(M)
-	if(!ismask(M))//FUCK YOU MASK OF NARNAR!
```

Comment:
you didn't write anything to replace this
add_to_hud(M) will never be called
pretty sure that's important